.github/workflows/ci.yml

```
@@ -63,7 +63,7 @@ jobs:
           # toolchain, components, etc. are specified in rust-toolchain.toml
 
       - name: Set up just
-        uses: ./.github/actions/setup-just
+        uses: $/.github/actions/setup-just
```

actionlint (1.7.12)

[error] specifying action "$/.github/actions/setup-just" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)